plugins/ImageReader/ImageReader.py

```diff
@@ -124,8 +128,15 @@ def _generateSceneNode(self, file_name, xz_size, peak_height, base_height, blur_
 
             Job.yieldThread()
 
-        height_data *= scale_vector.y
-        height_data += base_height
+        if use_transparency_model:
+            p = 1.0 / math.log(transmittance_1mm / 100.0, 2)
+            min_luminance = 2.0 ** ((peak_height - base_height) / p)
+            for (y, x) in numpy.ndindex(height_data.shape):
+                mapped_luminance = min_luminance + (1.0 - min_luminance) * height_data[y, x]
+                height_data[y, x] = base_height + p * math.log(mapped_luminance, 2)
```

Comment:
Should the log base not be `2.2` here?

Reply:
```
math.log(mapped_luminance, 2) * p
=
math.log(mapped_luminance, 2) / math.log(transmittance_1mm / 100.0, 2)
=
math.log(mapped_luminance) / log(2) / (math.log(transmittance_1mm / 100.0) / log(2) )
=
math.log(mapped_luminance) / math.log(transmittance_1mm / 100.0)
```
So the base of the log doesn't matter as long as it's the same in `p`.

I added a comment.